### ISSUE TITLE
Enabling Etherpad leads into error "URL constructor: /etherpad/p/ is not a valid URL."

### ISSUE THREAD

**Issue by tobru** [jitsi/docker-jitsi-meet#305]
When setting ETHERPAD_URL_BASE, the following error is shown in the webconsole of the browser:

```
2020-03-26T20:43:52.373Z [modules/xmpp/ChatRoom.js] <value>:  Error processing:etherpad node. TypeError: "URL constructor: /etherpad/p/ is not a valid URL."
```

Image tags used: `4101-2`

**Comment by codev** [comment]
I had to put the etherpad base url in config.js to get it to work, so I changed:

etherpad_base: '/etherpad/p/',

to

etherpad_base: 'https://myserver.com/etherpad/p/'

I'm not sure, maybe relative URLs work for http servers?

**Comment by saghul** [comment]
That fix is correct @codev , we have a bug in Jitsi Meet, which I hope to fix soon.

**Comment by saghul** [comment]
Fixed in https://github.com/jitsi/docker-jitsi-meet/commit/5e6faced4ff7724b65d608f6fe93fabe70743efd you can rebuild the containers and it should be working. See https://github.com/jitsi/docker-jitsi-meet/issues/311 a new release is coming up on monday.

**Comment by tobru** [comment]
Thanks a lot for taking care @saghul :+1:

**Comment by tele-tom** [comment]
That's not the solution, now we have this:

[JitsiMeetJS.js] <Object.getGlobalOnErrorHandler>:  UnhandledError: Uncaught SecurityError: Blocked a frame with origin "https://meet.mydomain.tld" from accessing a cross-origin frame. Script: https://meet.mydomain.tld/libs/app.bundle.min.js?v=3729 Line: 152 Column: 126770 StackTrace:  DOMException: Blocked a frame with origin "https://meet.mydomain.tld" from accessing a cross-origin frame.
    at c (https://meet.mydomain.tld/libs/app.bundle.min.js?v=3729:152:126770)
    at HTMLIFrameElement.t.onload (https://meet.mydomain.tld/libs/app.bundle.min.js?v=3729:152:127408)

**Comment by retani** [comment]
With the latest images (4101-2) and a full URL (`ETHERPAD_URL_BASE=http://etherpad.meet.jitsi:9001`), I get success in the console: 

`[modules/UI/UI.js] <Object.b.initEtherpad>:  Etherpad is enabled`

However, there is no button and I just don't understand how to access etherpad now.

**Comment by tele-tom** [comment]
Don't know what happened in the meanwhile, but now it also works for me with the config you mentioned...

**Comment by retani** [comment]
> Don't know what happened in the meanwhile, but now it also works for me with the config you mentioned...

Great! Do you know how the integration works? 😄 
I can see that it's running fine by itself, I can create a pad at https://jitsi.xxxxx.xx/etherpad/

Shouldn't the be a button in jitsi front end or anything at all to connect meet with etherpad? It does not seem to be connected to jitsi at all.

**Comment by tele-tom** [comment]
I have configured it in .env by just uncommenting the etherpad line.
`ETHERPAD_URL_BASE=http://etherpad.meet.jitsi:9001`

In ~/.jitsi-meet-cfg/web/config.js I set the URL to the full URL instead of the default '/etherpad/p'.
So this looks like
`etherpad_base: 'https://meet.mydomain.tld/etherpad/p/',`

Now I have an entry in the three dots menu called "Open shared document". When I click on it etherpad opens within an iframe.
